src/main/migrations/20211117081012-create-user.js

```diff
@@ -1,42 +1,58 @@
 'use strict';
 const { UUIDV4 } = require('sequelize');
 
+const users = [
```

Comment:
Instead of editing an existing migration, is it possible for you to create new ones to modify the tables? Generally for migrations, the standard is to make new ones when making additional edits inside the DB! 

More info here:
https://sequelize.org/v3/docs/migrations/
https://medium.com/@andrewoons/how-to-define-sequelize-associations-using-migrations-de4333bf75a7